docs/index.js

```diff
@@ -45,9 +45,10 @@ const NotationDisplay = function NotationDisplay(notationClass) {
   const span = document.createElement("span");
   inputContainer.after(span);
   return {
-    update(value) {
+    update(value, places) {
       const decimalValue = parse(value);
-      const formatted = decimalValue === null ? "???" : notation.format(decimalValue, 2, 0);
+      const decimalPlaces = parse(places);
```

Comment:
Should probably not be decimal, since some things rely on it being number (e.g., using +). I'd just do +places rather than parse(places), and rename the variable..